tools/github_webhook_processor.php

```diff
@@ -23,6 +23,9 @@
 //This is required as otherwise somebody could trick the script into leaking the api key.
 $hookSecret = '08ajh0qj93209qj90jfq932j32r';
 
+$prBalanceJson = '';	//Set this to the path you'd like the writable pr balance file to be stored, not setting it writes it to the working directory
+$featuresPerFix = 1;	//Number of allowed 'feature' PRs per merged 'fix' PR
```

Comment:
might be good in theory, but will probably just spam people. at least 2 would be better